app/code/Magento/Config/etc/system.xsd

```diff
@@ -95,6 +95,7 @@
                     <xs:element name="header_css" type="xs:string" />
                     <xs:element name="resource" type="typeAclResourceId" />
                     <xs:element ref="group" />
+                    <xs:element name="frontend_model" type="typeModel" />
```

Comment:
the attribute group for this element is defined on line 55, would it make more sense to put it there?